tests/test_operators.py

```diff
@@ -1,39 +1,31 @@
 from __future__ import annotations
 
-import pytest
-from typing import Callable
+from typing import List
```

Comment:
You can use `list` from annotations and remove it from here.

Reply:
done